docs/release-notes.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: docs
+title: "Release notes"
+permalink: /docs/release-notes
+---
+
+# Release notes
+
+## Microk8s 1.16 - 26 September 2019
+
+**MicroK8s** is a Kubernetes<sup>&reg;</sup> cluster delivered as a single snap
```

Comment:
Strange that only 1.16 has the full announcement while the rest of the releases have only the change logs.

Reply:
It isn't that strange - it seemed weird repeating essentially the same text each time - it just added extra words to the page so I removed the contact us section and the near identical opening paragraph